ivy/functional/ivy/device.py

```diff
@@ -264,7 +264,13 @@ def gpu_is_available() -> bool:
     """
     Determine whether a GPU is available to use, with the backend framework.
 
-    :return: Boolean, as to whether a gpu is available.
+    Returns
+    -------
+    out:
+        Boolean, as to whether a gpu is available.
+
+    Examples:
+    >>> print(ivy.gpu_is_available())
```

Comment:
also add an example output as done [here](https://github.com/unifyai/ivy/commit/57d97e94026dc81c66ba580112f2d72a9d405d59#diff-aa9310d758ed4fd979817c39c9df57480f958f3fc333d31ce99cdf750673a719R36) :)
A simple `True` would work

Reply:
Just one small change. I think `Examples` should be written like below following Numpy Doc String style:
```
Examples
--------
```
Rest LGTM 🙂

Reply:
Change made. Thanks for your help :)